_config.yml

```diff
@@ -29,11 +29,12 @@ google_analytics:
 defaults:
   - values:
       layout: default
-      payment_id: 7e6cdfc3-1276-44e9-9992-7cf4419750e1
-      transaction_id: ec2a9b09-601a-42ae-8e33-a5737e1cf177
-      payment_order_id: 09ccd29a-7c4f-4752-9396-12100cbfecce
-      merchant_id: 5cabf558-5283-482f-b252-4d58e06f6f3b
-      payment_token: afccf3d0016340620756d5ff3e08f69b555fbe2e45ca71f4bd159ebdb0f00065
-      api_host: api.externalintegration.payex.com
-      api_url: https://api.externalintegration.payex.com
+      paymentId: 7e6cdfc3-1276-44e9-9992-7cf4419750e1
+      transactionId: ec2a9b09-601a-42ae-8e33-a5737e1cf177
+      paymentOrderId: 09ccd29a-7c4f-4752-9396-12100cbfecce
+      merchantId: 5cabf558-5283-482f-b252-4d58e06f6f3b
+      paymentToken: afccf3d0016340620756d5ff3e08f69b555fbe2e45ca71f4bd159ebdb0f00065
+      apiHost: api.externalintegration.payex.com
+      apiUrl: https://api.externalintegration.payex.com
       front_end_url: https://ecom.externalintegration.payex.com
+      consumerSSN: 199710202392
```

Comment:
The naming of this variable does not conform to the rest of the system.
And is it used enough to warrant its own global variable?

Reply:
Changed format. It is used in 3 swish requests.. And I assume it can be used for invoice also. But if you think it is redundant I can remove it.